examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/TvCastingApp.java

```diff
@@ -232,7 +232,7 @@ private void reportSleepingVideoPlayerCommissioners(
     for (VideoPlayer player : cachedVideoPlayers) {
       Log.d(TAG, "Cached Video Player: " + player);
       // do NOT surface this cached Player if we don't have its MACAddress
-      if (player.getMACAddress() == null) {
+      if (player.getMACAddress() == null || player.getMACAddress().isEmpty()) {
```

Comment:
Should detail why we are surfacing cached Players with a MACAddress

Reply:
Done. added docs further down in the case where the player is actually reported.